### DISCUSSION TITLE
[Feature Request] Keep microphone always muted

### DISCUSSION THREAD

**Discussion by superm1991** [ChristopherKlay/StadiaEnhanced#219] [Ideas & Requests]
Hello,

Do you think it possible to offer a way to keep _"Who you're speaking to"_ to _**"None right now"**_?

![image](https://user-images.githubusercontent.com/4275183/151860785-f6bfcef4-efac-4515-b762-26ac18383ba5.png)

**Comment by wodnerpie** [comment]
I really think this is a good idea. I'm annoyed when I go online, Stadia tells me my microphone is open, so I have to open the overlay, go to audio, go to Microphone, select "None" and finally get back to the game

**Comment by ChristopherKlay** [comment]
It's hardly possible to auto-select this because the menu isn't loaded until you actually open it.

You *can* however go into the address bar, click the lock icon on the far left and disable your mic. It will still make you join voice chat in games (so you can hear others), but you simply won't be able to talk in them, since the mic is always disabled - which stays that way until you manually switch it on in the same menu again.
